protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
 
 The `body` of each message can be any of the following message types
 
-## `RequestForQuote`
-
+## `Offering`
+| field            | data type | required | description                                                                                          |
+| ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
+| `id` | string    | Y        | A unique identifier for this offering.|
+| `pair` | string    | Y        | The currency pair being offered, in the format of `basecurrency_countercurrency`.|
+| `unitPrice` | string    | Y        | Price of 1 unit of base currency denominated in counter currency.|
+| `fee`   | string       | Y        | Base fee associated with this offering |
+| `min`   | string       | Y        | Minimum amount of counter currency that the counterparty (Alice) must submit in order to qualify for this offering.|
+| `max`   | string       | Y        | Maximum amount of counter currency that the counterparty (Alice) can submit in order to qualify for this offering.|
+| `presentationRequest`   | PresentationRequest    | Y        |  PresentationRequest which describes the credential needed to choose this offer.|
+| `payinInstruments`   | list[PaymentInstrument]    | Y        |  A list of payment instruments the counterparty (Alice) can choose to send payment to the PFI from in order to qualify for this offering.|
+| `payoutInstruments`   | list[PaymentInstrument]    | Y        |  A list of payment instruments the counterparty (Alice) can choose to receive payment from the PFI in order to qualify for this offering.|
```

Comment:
What would it mean for a payout instrument to have a fee associated to it? 

Perhaps PayoutInstrument should be a separate object?

Reply:
i think payout instruments can have fees associated with it. for example, a wire transfer could have a fee for moving money out of the PFI's account?